docs/asf-docs/hyp3_mintpy_template.cfg

```diff
@@ -0,0 +1,9 @@
+mintpy.load.processor        = hyp3
+##---------interferogram datasets:
+mintpy.load.unwFile          = {DATA_DIR}/*/*unw_phase_clip.tif
+mintpy.load.corFile          = {DATA_DIR}/*/*corr_clip.tif
+##---------geometry datasets:
+mintpy.load.demFile          = {DATA_DIR}/DEM/dem_clip.tif
+##---------incAngleFile:
+mintpy.load.incAngleFile    =  {DATA_DIR}/*/*inc_map_clip.tif
+
```

Comment:
This file can be removed now that the config content have been moved into the notebook.

Reply:
file deleted